nc_py_api/ex_app/integration_fastapi.py

```diff
@@ -171,7 +172,7 @@ def __fetch_model_as_file(
 
             with builtins.open(result_path, "wb") as file:
                 last_progress = current_progress
-                for chunk in response.iter_bytes(5 * 1024 * 1024):
+                for chunk in response.iter_raw(-1):
```

Comment:
Niquests support `-1` as the chunk size. It's just plain better, it allows to take whatever arrives immediately. Skipping the whole buffer logic.